docs/fsharp/language-reference/compiler-messages/toc.yml

```diff
@@ -0,0 +1,15 @@
+- name: Compiler Errors and Warnings
```

Comment:
This would be the only area in the F# TOC that would have a sub-TOC? Is that by design?

Reply:
I'm copying the pattern wholesale from C# for this set of documents. That's the only reason I had for this layout.

Reply:
Sounds good. If this will expand dramatically, probably better to have it separate then,